remote-access/ssh/windows10.md

```diff
@@ -0,0 +1,33 @@
+# SSH using Windows 10
+
+You can use SSH to connect to your Raspberry Pi from a Windows 10 computer that is using *October 2018 Update* or later without having to use third-party clients.
+
+If you haven't already done so, go to Settings > Apps > Apps & features > Manage optional features > Add a feature, and choose to install *OpenSSH Client*.
+
+You will need to know your Raspberry Pi's IP address to connect to it. To find this, type `hostname -I` from your Raspberry Pi terminal.
+
+If you are running the Pi without a screen (headless), you can also look at the device list on your router or use a tool like `nmap`, which is described in detail in our [IP Address](../ip-address.md) document.
+
+To connect to your Pi from a different computer, copy and paste the following command into the terminal window, but replace `<IP>` with the IP address of the Raspberry Pi. Use `Ctrl + Shift + V` to paste in the terminal.
```

Comment:
Sorry for such a late follow-up, but is it correct to be talking about "the terminal window" here? (I've got no idea, I've never used Windows 10)

Reply:
Since the SSH feature is supported in both CMD and PowerShell, I presumed that "Terminal window" as a generic term would be suitable.